.github/workflows/test-dependents.yml

```diff
@@ -13,7 +13,23 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: install extra modules
-        run: cpm install -g Test::DependentModules
+        run: cpm install -g Test::DependentModules CPAN::Reporter
+             Math::Random::ISAAC Math::Random::ISAAC::XS MooseX::ConfigFromFile
+             MooseX::Getopt HTML::Tree HTML::Formatter Locale::Util RapidApp
+             App::perlimports Dist::Zilla PLS Promise::Me Archive::Tar::Wrapper
+             Term::ReadLine::Gnu Test::FailWarnings Test::File::ShareDir
+             Chess::Plisco Marpa::R2 Perl::Critic Test::HTML::Content
+             Test2::Plugin::Feature Chess::PGN::Parse Test::Base::Less
+             File::Next Test::Script Object::Method Perl::Critic::TooMuchCode
+             String::PerlQuote Test::Output Test::Simple XXX Chess::PGN::Parse
+             Data::Compare File::Edit::Portable Mock::Sub Test::Trap
+             File::Tempdir Mock::Sub File::Touch Hook::Output::Tiny
+             Module::Starter Test::CheckDeps File::PathList Readonly::XS
+             Statistics::Basic::Mean Statistics::Basic::Median
+             Statistics::Basic::StdDev Test::Compile Number::Format PPI::HTML
+             Pod::Elemental::Transformer::SynHi Text::HumanComputerWords
+             Test::BDD::Cucumber::StepFile Cucumber::TagExpressions
+             Test::WWW::Mechanize::Catalyst Devel::ebug::HTTP
```

Comment:
I think this would be more readable with one module per line, but overall it looks great.

What might be even more portable would be to put this list into a `cpanfile`. That would make it easier to run the test outside of a workflow as well. Then it would be be `cpm install -g --cpanfile path/to/cpanfile` if you wanted to pre-install the deps.

Reply:
Actually probably be even easier to use bits from Dependent test module to generate the cpanfile. :)

Reply:
Adjusted the flow such that all the requirements for the dependents are retrieved, stored into a cpanfile, and that then installed. No more hardcoding. :)